wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/tool/dockerimage/DockerimageToolProvisioner.java

```diff
@@ -128,6 +144,14 @@ public void provision(Devfile devfile, WorkspaceConfigImpl workspaceConfig)
     devfile.getTools().add(dockerimageTool);
   }
 
+  private EntryPoint toEntryPoint(MachineConfig machineConfig) throws WorkspaceExportException {
+    try {
+      return entryPointParser.parse(machineConfig.getAttributes());
+    } catch (InfrastructureException e) {
```

Comment:
Please check if `EntryPointParser#parse` throw an exception in case of error.

Reply:
It does in case of failure to parse the data in `EntryPointParser#parseAsList`

Reply:
There's even a test case for it :)